examples/bal-to-edi/bal-to-edi.md

```diff
@@ -0,0 +1,22 @@
+# Converting Ballerina records to EDI
+
+Same EDI schema and generated code used in Ballerina to EDI example can be used to convert Ballerina records of type SimpleOrder to EDI.
```

Comment:
"used in Ballerina to EDI example" is this correct? Or you meant to say EDI to ballerina example?